pkg/kyverno/common/common.go

```diff
@@ -372,6 +372,17 @@ func GetVariable(variablesString, valuesFile string, fs billy.Filesystem, isGit
 		for _, p := range values.Policies {
 			resourceMap := make(map[string]Resource)
 			for _, r := range p.Resources {
+				if val, ok := r.Values["request.operation"]; ok {
+					if val == "" {
+						r.Values["request.operation"] = "CREATE"
```

Comment:
Log line should also be here. Otherwise we only log in one of the two scenarios.